gravitee-apim-console-webui/src/management/api/runtime-logs-v4/api-runtime-logs.component.spec.ts

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { NoopAnimationsModule } from '@angular/platform-browser/animations';
+import { HttpClientTestingModule, HttpTestingController } from '@angular/common/http/testing';
+import { MatIconTestingModule } from '@angular/material/icon/testing';
+import { TestbedHarnessEnvironment } from '@angular/cdk/testing/testbed';
+import { InteractivityChecker } from '@angular/cdk/a11y';
+
+import { ApiRuntimeLogsModule } from './api-runtime-logs.module';
+import { ApiRuntimeLogsComponent } from './api-runtime-logs.component';
+import { ApiRuntimeLogsHarness } from './api-runtime-logs.component.harness';
+
+import { User as DeprecatedUser } from '../../../entities/user';
+import { CurrentUserService, UIRouterState, UIRouterStateParams } from '../../../ajs-upgraded-providers';
+
+describe('ApiRuntimeLogsComponent', () => {
+  let fixture: ComponentFixture<ApiRuntimeLogsComponent>;
+  let httpTestingController: HttpTestingController;
+  let componentHarness: ApiRuntimeLogsHarness;
+
+  const RUNTIME_LOGS_TAB_TITLE = 'Runtime Logs';
+  const SETTINGS_TAB_TITLE = 'Settings';
+
+  const initComponent = async () => {
+    const CURRENT_USER = new DeprecatedUser();
+
+    CURRENT_USER.userPermissions = ['api-log-r'];
+
+    const ROUTER_PARAMS: unknown = {};
+
+    const FAKE_UI_ROUTER = { go: jest.fn() };
+
+    TestBed.configureTestingModule({
+      imports: [NoopAnimationsModule, ApiRuntimeLogsModule, MatIconTestingModule, HttpClientTestingModule],
+      providers: [
+        { provide: UIRouterState, useValue: FAKE_UI_ROUTER },
+        { provide: UIRouterStateParams, useValue: ROUTER_PARAMS },
+        { provide: CurrentUserService, useValue: { CURRENT_USER } },
+      ],
+    }).overrideProvider(InteractivityChecker, {
+      useValue: {
+        isFocusable: () => true,
+      },
+    });
```

Comment:
Is it necessary on this component. Most of the time it is on components with dialogs for example. Can you remove it and check if it changes anything ?